portal/v2/src/ClusterDetailListComponents/Overview.tsx

```diff
@@ -82,71 +28,37 @@ const clusterDetailHeadings : IClusterDetails = {
-        super(props);
-    }
-    
-    public render() {
-        const headerEntries = Object.entries(clusterDetailHeadings)
-        const filteredHeaders: Array<[string, any]> = []
-        if (this.props.item.length != 0) {
-            headerEntries.filter((element: [string, any]) => {
-                if (this.props.item[element[0]] != null &&
-                    this.props.item[element[0]].toString().length > 0) {
-                        filteredHeaders.push(element)
-                    }
-            })
-            return (
-                <Stack styles={contentStackStylesNormal}>
-                    <Stack horizontal>
-                        <Stack id="Headers" styles={KeyColumnStyle}>
-                        {filteredHeaders.map((value: [string, any], index: number) => (
-                            <ClusterDetailCell style={KeyStyle} key={index} value={value[1]} />
-                            )
-                        )}
-                        </Stack>
-                        
-                        <Stack id="Colons" styles={KeyColumnStyle}>
-                        {Array(filteredHeaders.length).fill(':').map((value: [string], index: number) => (
-                            <ClusterDetailCell style={KeyStyle} key={index} value={value} />
-                            )
-                        )}
-                        </Stack>
-                        
-                        <Stack id="Values" styles={ValueColumnStyle}>
-                        {filteredHeaders.map((value: [string, any], index: number) => (
-                            <ClusterDetailCell style={ValueStyle}
-                            key={index}
-                            value={this.props.item[value[0]]} />
-                            )
-                        )}
-                        </Stack>
-                    </Stack>
-                </Stack>
-            );
-        } else {
-            return (
-                <Stack>
-                <Shimmer styles={headShimmerStyle} shimmerElements={headerShimmer} width="25%"></Shimmer>
-                {headerEntries.map(header => (
-                    <Shimmer key={header[0]} styles={ShimmerStyle} shimmerElements={rowShimmer} width="75%"></Shimmer>
-                    )
-                )}
-                </Stack>
-                )
-            }
-        }
-    }
\ No newline at end of file
+interface ClusterDetailItem {
+    key: number,
+    name: string,
+    value: string,
+}
+
+export function OverviewComponent(props: OverviewComponentProps) {
+    const items: ClusterDetailItem[] = useMemo(() => 
+        Object.entries(clusterDetailHeadings)
+            .map(([property, heading], index) => 
+                ({key: index, name: heading, value: props.item[property]})
+            ), 
+        [props.item])
+
+    return (
+        <Stack styles={contentStackStylesNormal}>
+            <Stack horizontal>
+                <ShimmeredDetailsList
```

Comment:
+1 For bringing in premade React components from [fluentui](https://github.com/microsoft/fluentui).